www/hosa04/SP2/script.js

```diff
@@ -0,0 +1,214 @@
+//pushed entered letters to an array
+function pushDataToArray() {
+    var inputText = $('#tip-window').val();
+    if (inputText.length <= 0) return;
+    inputText = inputText.toLowerCase();
+
+    guessedLettersArr.push(inputText);
+    $('#tip-window').val("");
+
+    console.log(guessedLettersArr);
+}
+
+
+//gets hang image based od mistakes
+function getHangImage() {
+    return ("hang" + wrongCount + ".png");
+}
+
+//changes the hang picture based on getHangImage
+function changeHangImage() {
+    $('#images-render').css("background-image", 'url(hang_images/' + getHangImage() + ')');
+}
+
+//basic variables
+var guessedLettersArr = [];
+var alphabet = "abcdefghijklmnopqrstuvwxyz";
+var offlineWords = ['bespin', 'hoth', 'coruscant', 'naboo', 'dagobah'];
+var unknown = [];
+var chosenWord = getRandomWord();
+var tries = 10;
+var correct = 0;
+var wrongCount = 0;
+var audio = new Audio('winner_music.mp3');
+
+
+//updates mistakes + tries
+function rewriteCounts() {
+    $("#mistakes").text("Mistakes: " + wrongCount);
+    $("#remaining").text("Remaining: " + (tries - wrongCount));
+}
+
+//refreshes the page
+function refreshPage() {
+    location.reload();
+}
+
+//deletes the page
+function deletePage() {
+    $("body").remove();
+}
+
+function removeDivs() {
+    $("#tip-window").remove();
+    $("#mistakes").remove();
+    $("#remaining").remove();
+    $("#secret").remove();
+    $("#images-render").remove();
+    $("#discovered").remove();
+    $("#refresh-button").remove();
+    $("#hint").remove();
+    $("a").remove();
+}
+
+//alert when you win + repeat?
+function winRepeatConfirmation() {
+    var txt;
+    if (confirm("YOU WON! :)" + "\n" + "Do you want to play again?")) {
+        refreshPage()
```

Comment:
pro novou hru se to nedela tak, ze proste refreshnes stranku, ale rucne resetnes pouzite prvky do vychoziho stavu pomoci javascriptu

treba ze udelas nove tlacitko na reset hry, kdy zaroven nactes nove slovo,